app/controllers/readings_controller.rb

```diff
@@ -70,17 +67,19 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
+    def authorize_reading
+      authorize @reading
+    end
```

Comment:
personal preference: I think it's cleaner to call directly `authorize @reading` on line 4, as opposed to defining a new method.

Reply:
Hmm I'm not sure I understand what you mean here. Hopefully I have some time to ask you tomorrow!

Reply:
Hmm, so you are defining a new method here called `authorize_reading` that returns `authorize @reading`. This saves 1 characters but I think add complexity to reading the code since now when I read line 4, I need to check the method you defined here to see what it does.

Reply:
Ah! ok I think I see what you mean. So it's better to just pass it in as ``authorize @reading`` directly into the methods because making it a method doesn't make it more readable?

Reply:
Yes, just my personal preference. I may be wrong